app/app.css

```diff
@@ -41,6 +41,52 @@ body {
     align-items: baseline;
 }
 
+/*social-network*/
+div.social-network {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 20px;
+    height: 60px;
+    margin-top: 8px;
+    text-align: center;
+    vertical-align: middle;
+    text-decoration: none;
+}
+
+a.social-network {
+    box-shadow: 0 7px 14px rgba(0, 0, 0, 0.25), 0 5px 5px rgba(0, 0, 0, 0.22);
+    text-decoration: none;
+    padding-top: 12px;
+    height: 60px;
+}
+
+a.social-network > div {
+    vertical-align: middle;
```

Comment:
`vertical-align` has no effect here (this only applies to *inline* and *table-cell* elements).